.jules/palette.md

```
@@ -0,0 +1,3 @@
+## 2024-05-23 - Center-align TUI help footer and add key hints
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)